templates/web/fixmystreet.com/footer-nav.html

```diff
@@ -1,24 +0,0 @@
-<div id="footer-nav">
```

Comment:
Removing this template means you can remove any location that includes it (which should only be base/footer.html but looks like that's been copied a few times elsewhere).